ldms/src/store/hello_stream/Plugin_l2_stream_csv_store.man

```diff
@@ -0,0 +1,151 @@
+.\" Manpage for Plugin_l2_stream_csv_store
+.\" Contact ovis-help@sandia.gov to correct errors or typos.
+.TH man 7 "6 Mar 2021" "v4" "LDMS Plugin l2_stream_csv_store man page"
+
+.SH NAME
+Plugin_l2_stream_csv_store - man page for the LDMS l2_stream_csv_store plugin
+
+.SH SYNOPSIS
+Within ldmsd_controller or a configuration file:
+.br
+config name=l2_stream_csv_store [ <attr>=<value> ]
+
+.SH DESCRIPTION
+With LDMS (Lightweight Distributed Metric Service), plugins for the ldmsd (ldms daemon) are configured via ldmsd_controller or a configuration file. The l2_stream_csv_store plugin is a DEVELOPMENTAL store that writes out either a single stream's data to csv format if the input type is a well-known json format or writes out the raw messages if the input type is str. Input type will be determined by the hello_cat_publisher or similar.
+
+
+.SH CONFIGURATION ATTRIBUTE SYNTAX
+.TP
+.BR config
+name=l2_stream_csv_store path=<path> container=<container> stream=<stream> [buffer=<0/1>] [rollover=<N>]
+.br
+configuration line
+.RS
+.TP
+name=<plugin_name>
+.br
+This MUST be l2_stream_csv_store.
+.TP
+path=<path>
+.br
+path to the directory of the csv output file
+.TP
+container=<container>
+.br
+directory of the csv output file
+.TP
+stream=<stream_a,stream_b>
+.br
+csv list of streams to which to subscribe.
+.TP
+buffer=<0/1>
+.br
+Optional buffering of the output. 0 to disable buffering, 1 to enable it with autosize (default)
+.TP
+rollover=<N>
```

Comment:
for reasons which as yet escape me, rollover never happens when configured with:

```
config name=l2_stream_csv_store path=${STOREDIR} container=csv stream=foo buffer=1 rollover=3
```

Reply:
The example and how to run it is provided in the next comment re ldms/scripts/examples.

Reply:
60 sec default to check for roll. extend your sequence